src/DocBuilder.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace SymfonyDocsBuilder;
+
+use Doctrine\RST\Builder;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Filesystem\Filesystem;
+use SymfonyDocsBuilder\CI\MissingFilesChecker;
+use SymfonyDocsBuilder\Generator\HtmlForPdfGenerator;
+use SymfonyDocsBuilder\Generator\JsonGenerator;
+
+final class DocBuilder
```

Comment:
Unless I'm missing something, this class isn't used anywhere IN the bundle. It's just a public class that can be consumed by any project. If we DO decide to keep the `BuildDocsCommand`, I guess that command should also (in the future) use this class.

Reply:
As far as tests go, if this is the central mechanism to build the docs, then this could/should be used in the IntegrationTest :)